src/main/java/seedu/algobase/logic/commands/SwitchTabCommand.java

```diff
@@ -0,0 +1,62 @@
+package seedu.algobase.logic.commands;
+
+import static seedu.algobase.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.algobase.logic.parser.CliSyntax.PREFIX_TAB_INDEX;
+import static seedu.algobase.logic.parser.CliSyntax.PREFIX_TAB_TYPE;
+
+import seedu.algobase.commons.core.index.Index;
+import seedu.algobase.logic.commands.exceptions.CommandException;
+import seedu.algobase.model.Model;
+import seedu.algobase.model.gui.TabEnum;
+
+/**
+ * Switch tabs in the GUI.
+ */
+public class SwitchTabCommand extends Command {
```

Comment:
What's the reason for changing `switch` to `switchtab`?

Reply:
To package the commands under the 'tab' feature